Felienne/hedy
# [Improvement] Return warning of user overwrites saved program

**Description**
Currently users can store programs which are identified by their name. When saving a program with an identical name to a previously saved program the older version is overwritten without a warning. This is undesirable as this might accidentally overwrite a submitted program (automatically losing the submit status!). In this PR we implement a fix to return the user an additional confirmation pop-up if they are about to overwrite an already existing program.

**Fix for**
This PR fixed #1433.

**How to test**
Save a program with and without an already existing saved program name. Notice the additional pop-up to confirm saving the program with occur.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
